main.go

```diff
@@ -31,6 +31,10 @@ func main() {
 	app.Usage = ""
 	app.Version = Version
 	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "helmcommand, c",
+			Usage: "overwrite helm command. Uses 'helm by default",
```

Comment:
`path to helm binary`, and no need to note about the default value. AFAIK cobra include it in the usage when you set the default value via the `Value` field.